docs/features/models/few_shot_classification.ipynb

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
Update structure similarly to https://github.com/deeppavlov/DeepPavlov/pull/1644
Also check again file content. You show evaluate command, but dnnc_infer now doesn't support evaluation, since it has no train section.